docs/sql/sql.md

```diff
@@ -0,0 +1,440 @@
+# SQL Support
+
+SpacetimeDB supports two subsets of SQL.
+One for ad hoc queries against the database.
+These can be made through the [cli] or the [http] endpoint.
+Another for subscriptions made through the [websocket] api or [sdk].
+
+## Subscriptions
+
+```ebnf
+SELECT projection FROM relation [ WHERE predicate ]
+```
```

Comment:
Note that previously the syntax was described using syntax diagrams stored in the [spacetime-web](https://github.com/clockworklabs/spacetime-web) repo. The `ebnf` specification makes this document easier to review, but probably less consumable from a UX perspective.

Reply:
I actually think it might be better UX. As an anecdote, I find these easier to reason about.

Reply:
It's your call. Should we include both?

Reply:
Just keep `ebnf` only.